libsql-wal/Cargo.toml

```diff
@@ -19,3 +19,12 @@ thiserror = "1.0.58"
 tracing = "0.1.40"
 walkdir = "2.5.0"
 zerocopy = { version = "0.7.32", features = ["derive", "alloc"] }
+tokio = { version = "1", features = ["full"] }
```

Comment:
do we need all the features?

Reply:
Probably not, but we're compiling them anyway. I'll trim later.